src/components/Dashboard/TodoList.vue

```diff
@@ -416,7 +415,7 @@
 
                   <!-- dropdown menu -->
                   <v-menu
-                    v-if="!isCancellableTodoItem(item)"
+                    v-if="isCancellableTodoItem(item)"
```

Comment:
Did you mean to change this?

Reply:
Yes Sev!

Although the previous code worked, I was a bit weird to read and understand because of negations. Also, the function's name didn't actually resemble what it was doing.

I just made the change to simplify the logic and make it easier to understand.